template/src/components/PointsOfInterest/PointOfInterest.tsx

```diff
@@ -30,11 +34,16 @@ const PointOfInterest: FC<PointOfInterestProps> = ({ model, onDelete, ...other }
     useWatchAndRerender(model, ["title", "geometry.x", "geometry.y", "color"]);
     const [lon, lat] = xyToLngLat(geometry.x, geometry.y);
     return (
-        <MenuItem {...other}>
-            <div className="PointOfInterest-swatch" style={{ backgroundColor: color.toCss() }} />
+        <MenuItem button={true} {...other}>
```

Comment:
nit: `<MenuItem button={true}>` is equivalent to `<MenuItem button>`.

Reply:
Although now i realize that `button` shouldn't be needed at all as it's the default value for `MenuItem`.

Reply:
I am used to webs linting rules :) updated